### PR TITLE
Day two iteration

### PR DESCRIPTION
**Pull request by fluty84**

### CODE REVIEW

**Comment by fluty84** [comment]
I have chosen to handle messageapp errors and return them to the client.

File: README.md [hunk 1]
```diff
@@ -1,77 +1,26 @@
...
-$ docker-compose up
-```
-If you use linux, that should work out of the box, but remember that if you're using mac, you need to start the docker daemon first, or else you'll get an error on the console.
+- /messages 
+  - Method POST => 
+  - Response: 200 
+  - Error responses:
 
-The output of the command will show it downloads the image and spins up the container with the service. Once the service has started, you can check that it works correctly by sending a request. You can use whatever tool you are comfortable with, such as Postman or cURL, for instance. Try this request:
+     - 427 "Need destination & message keys"
```
Comment (joskar) [inline README.md]: Why did you choose this status code?
let's say that you are the consumer of this error.
What will happen when you search http 427 status code ?

File: routes/messages.routes.js [hunk 2]
```diff
@@ -16,7 +16,26 @@ router.post("/", (req, res) => {
     MessageAppService
         .saveMessage(destination, body)
         .then(response => res.json(response.data))
-        .catch(err => res.json(err))
+        .catch(err => {
+            
+            if(!err.config.data.includes("destination") && !err.config.data.includes("body") ){
```
Comment (joskar) [inline routes/messages.routes.js:35]: Why do you check the data after you send ?
Why do you use err.config.data?
Is this some axios magic can you explain?

File: README.md (hunk 1 above)
Comment (fluty84) [reply README.md]: Sorry I did't know, that there are predefined codes... 
I will search for the correct ones!!

File: routes/messages.routes.js (hunk 2 above)
Comment (fluty84) [reply routes/messages.routes.js:35]: Of course I can do it too! 

err.config.data responses with the payload that the service messageapp recieve, and I use that to make a response to the user

**Pull request merged**